man/virt-install.rst

```diff
@@ -1338,6 +1338,15 @@ Some example suboptions:
 
     For ``HOSTDEV`` format, see ``--hostdev`` documentation
 
+``portForward=[ADDRESS:]GUESTPORT:HOSTPORT[/PROTO]``
```

Comment:
I believe this should be `[ADDRESS:][HOSTPORT:]GUESTPORT[/PROTO]`.

The host part definitely comes first, otherwise `8080:80` wouldn't make much sense: you'd be mapping an unprivileged guest port to a privileged host one, defeating the purpose of using unprivileged guest networking in the first place.

And the `HOSTPORT` should be marked as optional, since libvirt only requires the host part (`start`) to be provided unless the guest part (`to`) doesn't match. This would be *almost* consistent with how the corresponding docker/podman feature works.

I say "almost" because podman/docker would react to `-p 80` by mapping guest port 80 to a random unprivileged host port, whereas libvirt would react to `portForward=80` by attempting to map guest port 80 to... Host port 80, which is probably not what you wanted and will likely fail.

That is not virt-manager's problem though.

Reply:
Sounds good, will fix!